tests/security/audit/aureport.pm

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "aureport" utility can generate default audit report and reports with specifc parameters
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $audit_log = '/var/log/audit/audit.log';
+
+    select_console 'root-console';
+
+    # Make sure audit service is started
+    assert_script_run('systemctl is-active auditd');
+
+    # Generate audit records for testing
+    assert_script_run("echo '' > $audit_log");
+    assert_script_run('systemctl stop apparmor');
+    assert_script_run('systemctl start apparmor');
+
+    # Stop auditd before checking records in case the following executions will generate new audit records
+    assert_script_run('systemctl stop auditd');
+
+    # Generate report by default
+    assert_script_run('aureport');
+
+    # Generate AVC report
+    validate_script_output('aureport -a', sub { m/AVC/ });
+
+    # Compare number of AVC reports is the same as it is shown in default report
+    my $default_num = script_output("aureport | grep 'AVC' | cut -d ' ' -f 4");
+    my $tmp_output = '/tmp/output';
+    my $tmp_backup = '/tmp/backup';
+    assert_script_run("aureport -a > $tmp_output");
+    assert_script_run("tail -1 $tmp_output > $tmp_backup");
+    my $avc_num = script_output("cat $tmp_backup | cut -d '.' -f 1");
+    if ($default_num == $avc_num) {
+        record_info('Comparison passed: ', 'AVC reports counted in default report is the same as generated by AVC reports');
+    } else {
+        record_info('Error: ', 'AVC reports counted in default report do not match the counts in AVC reports');
+    }
```

Comment:
it should be failed here

Reply:
Result 'fail' added along with tc tags for previous cases.